src/Auth0Client.ts

```diff
@@ -493,4 +488,89 @@ export default class Auth0Client {
     const url = this._url(`/v2/logout?${createQueryParams(logoutOptions)}`);
     window.location.assign(`${url}${federatedQuery}`);
   }
+
+  private async _getTokenFromIFrame(
+    options: GetTokenSilentlyOptions
+  ): Promise<any> {
+    const stateIn = encodeState(createRandomString());
+    const nonceIn = createRandomString();
```

Comment:
@stevehobbsdev 👆

Reply:
Also ... why are we even doing anything with `state` here?

Reply:
This is part of the functionality that gets a new token using an iframe. The state and nonce values are passed through the `/authorize` URL. Even though we're using `prompt=none`, I assume we'd still have to pass `state` and `nonce` through?

As for why nonce is not encoded, I'm not sure. This is code that existed before the RT work so I don't have the background on that decision. Looking at it, it uses the same function `createRandomString` that's used to generate the state so it would make sense that `nonce` is also encoded. Let me look into changing that.

Reply:
`nonce` is now encoded wherever it is used.